snuba/settings/__init__.py

```diff
@@ -135,6 +135,9 @@
 
 MAX_RESOLUTION_FOR_JITTER = 60
 
+# These contexts will not be stored in the transactions table
```

Comment:
can you write an example in the comment of what the configuration would be / look like / represent?

Reply:
done